templates/faq.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block content %}
+
+<!-- help needed -->
+<!-- needs to be handled from backend urls and views -->
```

Comment:
@JimLynx 
Looks like you've coded in the backend urls/views now? Can this be removed?